src/ws/Makefile-ws.am

```diff
@@ -159,12 +159,20 @@ remotectl_LDADD = \
 	$(COCKPIT_LIBS) \
 	$(NULL)
 
-dist_systemdunit_DATA += src/ws/cockpit.socket
-nodist_systemdunit_DATA += src/ws/cockpit.service
+nodist_systemdunit_DATA += \
+	src/ws/cockpit.socket \
+	src/ws/cockpit.service \
+	$(NULL)
 
 firewalldir = $(prefix)/lib/firewalld/services
 firewall_DATA = src/ws/cockpit.xml
 
+issuedir = $(sysconfdir)/$(PACKAGE)/
```

Comment:
Do we really want to treat these as configuration in /etc? The standard issue and motd files are in /usr nowadays, to better support stateless systems. Also, configuration files are rather awkward to deal with on upgrades.

Reply:
My thinking was that having them in configuration allows users to adjust the messages on stateless systems. Makes things more easily configurable.

But we can move them I'm open to putting them somewhere else if you think that's better